cmake/modules/AddSwift.cmake

```diff
@@ -2423,7 +2423,7 @@ function(add_swift_host_tool executable)
   add_dependencies(${ASHT_SWIFT_COMPONENT} ${executable})
   swift_install_in_component(TARGETS ${executable}
                              RUNTIME
-                               DESTINATION bin
+                               DESTINATION "${ASHT_DESTINATION_PREFIX}bin"
```

Comment:
I'm not entirely sure if this is an idiomatic enough way to compose path in CMake -- I've tried exploring `get_filename_component` and `file(TO_NATIVE_PATH )`, but none of those seems to fit this scenario (prepend a folder to a relative path)

Reply:
Can we have a boolean parameter called `IS_LOCAL` rather than asking the called to pass down the prefix? this may lead to the duplications of the prefix.

Reply:
Sure -- I've made this change in d70625f820a7475093c2fcd0baae72016077c9b4

Reply:
I really am against the idea of adding more parameters to `swift_install_in_component`.  This function is needlessly complicated and deviates from the standard practices for CMake, making this codebase more difficult for new comers.  I think that it is better to use this as CMake does, and we can absolutely just use `local/bin` as the destination.